src/Plugin/facets/url_processor/DgiIrCustomizationsQueryString.php

```diff
@@ -168,6 +164,7 @@ public function buildUrls(FacetInterface $facet, array $results) {
           $new_url_params['search_api_fulltext'] = '';
         }
 
+
```

Comment:
nit picking like a casual nit-picker. Bonus new line?

Reply:
copy paste from the lovely folks at the facets module